build_tools/circle/build_doc.sh

```diff
@@ -51,7 +51,7 @@ get_build_type() {
     fi
     git_range="origin/master...$CIRCLE_SHA1"
     git fetch origin master >&2 || (echo QUICK BUILD: failed to get changed filenames for $git_range; return)
-    filenames=$(git diff --name-only $git_range)
+    filenames=$(git diff --name-only $git_range -- `git ls-files | grep -E "/plot_|.*rst$"`)
```

Comment:
I'm still unable to understand why we should filter files here.
I'd propose something like
```
filenames=$(git diff --name-only $git_range
changed_examples=$(echo "$filenames" | grep -e ^examples/plot_)
```
and why do we need to handle .rst files here？

Reply:
fair enough. Done!